### ISSUE TITLE
 loss/accuracy different between bvlc/caffe with NVIDIA/caffe

### ISSUE THREAD

**Issue by zlheos** [NVIDIA/caffe#532]
Hello, I have some questions when using NVIDIA/caffe, and hope you can help me.
finetune model：
with bvlc/caffe , the accuracy is always 0.97
with NVIDIA/caffe , the accuracy drop from 0.98 to 0.58 and then restore to 0.7 

prototxt and log is below

I don't know why 
[train_prototxt.txt](https://github.com/NVIDIA/caffe/files/2317600/train_prototxt.txt)
[run_bvlc.log](https://github.com/NVIDIA/caffe/files/2317601/run_bvlc.log)
[run_nvidia.log](https://github.com/NVIDIA/caffe/files/2317602/run_nvidia.log)

@drnikolaev

**Comment by rotorliu** [comment]
I have the same question.
Same model prototxt
The loss of NVIDIA Caffe is more than the loss of BLVC Caffe when training.
Why?

**Comment by drnikolaev** [comment]
@zlheos @rotorliu Please use NVCaffe-specific implementation of BatchNorm layer.
